recipes/libpng/all/conanfile.py

```diff
@@ -174,6 +182,8 @@ def package_info(self):
 
         prefix = "lib" if self._is_msvc else ""
         suffix = "d" if self.settings.build_type == "Debug" else ""
-        self.cpp_info.libs = ["{}png16{}".format(prefix, suffix)]
+        major_min_version = f"{Version(self.version).major}{Version(self.version).minor}"
```

Comment:
Could this have broker libpng, the lib in show `libpng<bound method Version.major of '1.6.37'><bound method Version.minor of '1.6.37'>`, should that have been function calls?

Reply:
@gummif update your Conan client. It was a bug, but is fixed on 1.50.2 and 1.51.2